packages/ui-kit/src/components/code-block.js

```diff
@@ -201,15 +240,51 @@ const languageAliases = {
   terminal: 'bash',
   js: 'javascript',
 };
+const languageDisplayNames = {
```

Comment:
I'm not sure I like this mapping, it looks "fragile". A lot of languages are missing and we should avoid updating this map every time we are missing a language.

We should check if there is an existing library or something where we can pull the list from. Less manual work, the better and more maintainable it is.

Reply:
I will move this into the transformer plugin because it's actually content logic and not UI logic. 

concerning missing languages: We're building _our_ documentation stack here not a product that has to serve any case in the world.  If a community picks up using our kit it's trivial PRs to add more from what they actually use.